tests/_support/Module/SprykHelper.php

```diff
@@ -329,4 +330,20 @@ protected function getContainer(): ContainerInterface
 
         return $symfony->_getContainer();
     }
+
+    /**
+     * @param \Codeception\Test\Unit $testClass
+     * @param string $class
+     * @param int $expectedNumberOfCalls
+     *
+     * @return void
+     */
+    public function mockCommand(Unit $testClass, string $class, int $expectedNumberOfCalls): void
```

Comment:
This method is particular to get a mock with an expected number of calls to a method. I would change it to the following

```suggestion
    public function getMockWithExpectedNumberOfMethodCalls(string $class, string, $method, int $expectedNumberOfCalls = 1): void
```
TBH I never used to pass the testCase to the Stub, what is the reason for doing it?

Reply:
Renamed. Without passing a testCase, fewer numbers of calls method than specified will not cause an error.